tasks/page/run_ibom

```diff
@@ -5,32 +5,42 @@ import sys
 import os
 import os.path
 import wx
+import pcbnew
 
 from InteractiveHtmlBom.InteractiveHtmlBom.core import ibom
 from InteractiveHtmlBom.InteractiveHtmlBom.core.config import Config
 from InteractiveHtmlBom.InteractiveHtmlBom.ecad import get_parser_by_extension
+from InteractiveHtmlBom.InteractiveHtmlBom.ecad.kicad import PcbnewParser
 from InteractiveHtmlBom.InteractiveHtmlBom.version import version
 
 
-kicad_file = os.path.abspath(sys.argv[1])
+board_file = os.path.abspath(sys.argv[1])
 info_file = os.path.abspath(sys.argv[2])
 out_file = os.path.abspath(sys.argv[3])
 
 BASEDIR = os.path.dirname(sys.argv[0])
 IBOM_CONFIG = os.path.join(BASEDIR, 'ibom_config/config.ini')
 
-os.chdir(os.path.dirname(kicad_file))
+os.chdir(os.path.dirname(board_file))
 
 logger = ibom.Logger(cli=True)
 ibom.log = logger
 
+board = None
+ext = os.path.splitext(board_file)[1]
+if ext == '.brd':
+    board = pcbnew.LoadBoard(board_file, pcbnew.IO_MGR.EAGLE)
```

Comment:
This is what breaks the offsets for some graphical items. Save the board after conversion and then reload it from file.

See https://github.com/openscopeproject/InteractiveHtmlBom/issues/193#issuecomment-705799746

Reply:
Thanks! I'll do that.

Reply:
I think this looks better now. It's still not perfect (some of the text sizes are off), but it's pretty good. Here's a comparison between the Tracespace Gerber view and the IBOM view:

![2020-10-10-131527_1914x1050_scrot](https://user-images.githubusercontent.com/1113720/95653741-3eb43a80-0afb-11eb-8c59-03e43d7f1299.png)
![2020-10-10-131523_1914x1050_scrot](https://user-images.githubusercontent.com/1113720/95653740-3d830d80-0afb-11eb-875b-eac56977ed8f.png)

Reply:
Font sizes is issue of the eagle importer. They look a bit off in kicad itself

![image](https://user-images.githubusercontent.com/3622532/95653990-a8a7f200-0ab1-11eb-9d0a-186ca122e22a.png)

Reply:
I think those parts are down to the KiCad import. The writing is also off in KiCad itself (see the [screenshots above](https://github.com/kitspace/kitspace/pull/300#issuecomment-684946674)). It's probably a font issue :thinking:. Either way, it's probably as good as we can get it for now then.